src/PaymentGateways/Gateways/Stripe/Actions/GetStripeGatewayData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe\Actions;
+
+use Give\Donations\Models\Donation;
+use Give\Framework\Exceptions\Primitives\Exception;
+use Give\Framework\PaymentGateways\Log\PaymentGatewayLog;
+use Give\PaymentGateways\Gateways\Stripe\ValueObjects\PaymentMethod;
```

Comment:
Should we add an alias that further communicates that this is the Give Value Object? Something like `LegacyPaymentMethodValueObject`?

I think it would be helpful to avoid confusion when this is re-visited.

Reply:
Good idea, i'll update that